src/pages/library/DataBrowser.js

```diff
@@ -87,7 +87,7 @@ const allColumns = {
   consortiums: { title: 'Consortiums', contents: row => _.join(', ', getConsortiumsFromDataset(row)) },
   subjects: { title: 'No. of Subjects', contents: row => row?.counts?.donors },
   dataModality: { title: 'Data Modality', contents: row => _.join(', ', row.dataModality) },
-  lastUpdated: { title: 'Last Updated', contents: row => row.lastUpdated ? Utils.makeStandardDate(row.lastUpdated) : null },
+  lastUpdated: { title: 'Last Updated', contents: row => !!row['dct:modified'] ? Utils.makeStandardDate(new Date(row['dct:modified'])) : null },
```

Comment:
Shouldn't this be done in a method, so it can be used in the details page as well as in a column? Also we may want a date facet at some point.

Reply:
I figured we would make those changes when we needed them. Since this is only done in the one place right now, I figured we would inline it.

I think ultimately doing it in a method is probably correct, long term.

Reply:
Ah, actually it was being done in two places, just only one place used the field on the dataset. Updated.